boards/arm/actinius_icarus/dts/bindings/sim-select.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  The Icarus boards provide the user with 2 options regarding SIM usage for
+  LTE-M/NB-IoT communication. Either using the on-board eSIM or using an
+  external nano-SIM. The SIM can be selected using the SIM select pin given by
+  the sim-gpios property by setting it to 0 (eSIM) or to 1 (external nano-SIM).
+
+compatible: "sim-select"
```

Comment:
should we use some kind of board-specific prefix here? like `icarus,sim-select` ?

Reply:
ok, NVM, just saw https://github.com/zephyrproject-rtos/zephyr/pull/43817#pullrequestreview-916458049 . I thought we were going in the opposite direction.